setup.py

```diff
@@ -46,8 +46,9 @@
         "nbformat>=5.0,<5.6",
         "numpy>=1.19",
         "pandas>1.0",
+        "pulp==2.3.1",
```

Comment:
Weird Snakemake issue. We should look into upgrading to Snakemake v8 soon. In any case, we don't need to keep this in the setup.py long term.